src/main/java/net/snowflake/ingest/utils/ParameterProvider.java

```diff
@@ -67,8 +67,8 @@ public class ParameterProvider {
       Constants.BdecParquetCompression.GZIP;
 
   /* Iceberg mode parameters: When streaming to Iceberg mode, different default parameters are required because it generates Parquet files instead of BDEC files. */
-  public static final int MAX_CHUNKS_IN_BLOB_AND_REGISTRATION_REQUEST_ICEBERG_MODE_DEFAULT =
-      1; // 1 parquet file per blob
+  public static final int MAX_CHUNKS_IN_BLOB_AND_REGISTRATION_REQUEST_ICEBERG_MODE_DEFAULT = 1;
+  public static final long MAX_CLIENT_LAG_ICEBERG_MODE_DEFAULT = 30000;
```

Comment:
Could you move this to the "lag related parameters" section?

Reply:
Moved.